get_source.sh

```diff
@@ -57,7 +64,23 @@ for i in "$@" ; do
 			usage
 			;;
 
-		-openj9-repo=* | -openj9-branch=* | -openj9-sha=* | -openj9-reference=* | -omr-repo=* | -omr-branch=* | -omr-sha=* | -omr-reference=* | -parallel=* )
+	      -gskit-bin=* \
```

Comment:
This line should be identical to the [corresponding line in the head stream](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/get_source.sh#L67).

Reply:
updated. (It seems to be my IDE problem)